ch01.asciidoc

```diff
@@ -32,35 +32,35 @@ Due to the used onion routing which is also used in tor even the nodes that are
 
 As an electronic cash system preserves the 3 important properties of money (medium of exchange, store of value, and unit of account).
 The invention of money (and in particular Bitcoin) was primarily made to facilitate trade and enable the exchange of value between people.
-However without the Lightning Network Bitcoin is hard to be used concurrently by millions of people.
+However, without the Lightning Network Bitcoin is hard to be used concurrently by millions of people.
 Therefore, in order to fully understand the uses of the Lightning Network, we'll examine it from the perspective of people using it.
 In particular the use cases will come from previous users of Bitcoin as well as people who have not used Bitcoin before.
 Each of the people and their stories, as listed here, illustrates one or more specific use cases.
 We'll be seeing them throughout the book:
 
 consumer::
-A regular consumer on the Internet or in the offline world who wants to do purchases
+A regular consumer on the Internet or in the offline world who wants to make purchases
 
 content creator / curator::
 A person or platform offering content on the web.
-They want to install a paywall or get tipped by their fans and consumers.
+They want to install a payroll or get tipped by their fans and consumers.
```

Comment:
It is supposed to be pay wall and not payroll

Reply:
It seems that I have successfully modified it.
Sorry, I am not very familiar with git, I am still studying hard.